packages/fe-container/src/composer/NodeCreator.ts

```diff
@@ -54,18 +54,18 @@ const retrieveEventBus = (configuration: Configuration): Subject<any> | undefine
   return eventBus
 }
 
-const createEnrichedElement = (configuration: Configuration, defaultEventBus: Subject<any>) => {
+const createEnrichedElement = (configuration: Configuration, currentUser: object, defaultEventBus: Subject<any>) => {
```

Comment:
Define the type, is not a plain object

Reply:
type here would be a type exported by microlc.. to keep it as general as possible i can either keep it this way or maybe use Record<string,unknown> as a type. Is the latter you recommend?

Reply:
You should not be keep it as general as possible, because as you said is already defined in microlc.

We could publish the core package as npm module, and I think I'll do it when I'll come back, but for now copy the same definition here

Reply:
ok.. then I'll use the User.ts types from microlc

Reply:
Here is still object, you should use a partial user